### PR TITLE
AM62AX ThreadX Example Documentation and Fixes

### PR DESCRIPTION
**Pull request by jblanchard-jblopen**
Add ThreadX Documentation for the ThreadX. FileX and NETX examples.
Fix build warning in filex example.

### CODE REVIEW

File: docs_src/docs/api_guide/components/networking/networking.md [hunk 1]
```diff
@@ -116,7 +116,10 @@ Ethernet Low-Level Driver (\ref ENET_LLD) is a driver that aims at providing an
 
 
 ### TCP/IP Stack
--\subpage NETWORKING_LWIP
```
Comment (shiva-ti) [inline docs_src/docs/api_guide/components/networking/networking.md:119]: we shouldn't remove any existing section. Any reason why LWIP section is removed ?

**Review by shiva-ti — CHANGES_REQUESTED** [review]

**Review by shiva-ti — CHANGES_REQUESTED** [review]
Documentation added for ThreadX should have a "THREADX" specific tag in addition to "SOC_AM62AX", this can be added in "ENABLED_SECTIONS" in file "docs_src/docs/api_guide/device/am62ax/includes.cfg"
This "THREADX" section can be enabled by default, but if needed we should be able disable threadX specific docs from main mcu_plus_sdk docs.

**Comment by jblanchard-jblopen** [comment]
Hi,

Thank you for your feedback. We will be looking at it today.

Jonathan

**Comment by shiva-ti** [comment]
Merging the PR, the comments can be addressed as additional commit later

**Pull request merged**